content/minfraud/release-notes/2022.mdx

```diff
@@ -7,6 +7,23 @@ title: minFraud Release Notes
 Subscribe to the [minFraud release notes RSS feed](/minfraud/release-notes/rss.xml).
 </Alert>
 
+<ReleaseNote date="2022-03-01" title="Expanded filtering options in the account portal for minFraud transactions">
+The transaction review screen in your MaxMind account portal now has expanded
+options to filter your transactions. You may now filter transactions by:
+
+- **account user ID:** the user ID of the end-user who initiated the transaction
```

Comment:
We might want to capitalize the first letter of each of these to match the casing on the minFraud transactions page.